src/Core/Entity/Notification.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Ushahidi Notification Entity
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Platform
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\Core\Entity;
+
+use Ushahidi\Core\StaticEntity;
+
+class Notification extends StaticEntity
+{
+
+	protected $id;
+	protected $contact_id;
+	protected $set_id;
+	protected $is_subscribed;
+	protected $created;
+
+	// DataTransformer
+	protected function getDefinition()
+	{
+		return [
+			'id'            => 'int',
+			'contact_id'    => 'int',
```

Comment:
I think we should also have aliases for `contact` and `set` here.. see what we do with foreign keys on other Entities